### PR TITLE
Add version to QTI Binary Storage.

### PR DESCRIPTION
**Pull request by bugalot**
This PR aims at providing the QTI Binary Storage format with a version number. This will enable us to extend the QTI Binary Storage in the future, if required.

### CODE REVIEW

File: src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php [hunk 1]
```diff
@@ -53,6 +53,8 @@
  */
 abstract class AbstractQtiBinaryStorage extends AbstractStorage
 {
+    const VERSION = 1;
```
Comment (julien-sebire) [inline src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php]: Shouldn't we apply https://semver.org/ here?

File: src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php [hunk 2]
```diff
@@ -234,6 +239,9 @@ public function retrieve($sessionId)
             $stream->open();
             $access = $this->createBinaryStreamAccess($stream);
 
+            // -- Consume version (not used yet).
```
Comment (julien-sebire) [inline src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php]: May we expect issues when retrieving sessions that were saved without version?

File: src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php (hunk 1 above)
Comment (bugalot) [reply src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php]: I would say it's overkill for a binary protocol

File: src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php (hunk 2 above)
Comment (bugalot) [reply src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php]: Yes, we will release this as a breaking change.

**Pull request merged**